CHANGELOG.md

```diff
@@ -1,3 +1,92 @@
+  - Temporal.DateTime.toMonthDay -> Temporal.PlainDateTime.toPlainMonthDay
+  - Temporal.DateTime.toTime -> Temporal.PlainDateTime.toPlainTime
+  - Temporal.DateTime.toYearMonth -> Temporal.PlainDateTime.toPlainYearMonth
+  - Temporal.Duration.minus -> Temporal.Duration.subtract
+  - Temporal.Duration.plus -> Temporal.Duration.add
+  - Temporal.Instant.difference -> Temporal.Instant.since
+  - Temporal.Instant.getEpochMicroseconds (method) -> Temporal.Instant.epochMicroseconds (property)
+  - Temporal.Instant.getEpochMillieconds (method) -> Temporal.Instant.epochMillieconds (property)
+  - Temporal.Instant.getEpochNanoseconds (method) -> Temporal.Instant.epochNanoseconds (property)
+  - Temporal.Instant.getEpochSeconds (method) -> Temporal.Instant.epochSeconds (property)
+  - Temporal.Instant.minus -> Temporal.Instant.subtract
+  - Temporal.Instant.plus -> Temporal.Instant.add
+  - Temporal.MonthDay -> Temporal.PlainMonthDay
+  - Temporal.MonthDay.toDateInYear -> Temporal.MonthDay.toPlainDate
+  - Temporal.now.time -> Temporal.now.plainTimeISO
+  - Temporal.Time -> Temporal.PlainTime
+  - Temporal.Time.difference -> Temporal.PlainTime.since
+  - Temporal.Time.minus -> Temporal.PlainTime.subtract
+  - Temporal.Time.plus -> Temporal.PlainTime.add
+  - Temporal.Time.toDateTime -> Temporal.PlainTime.toPlainDateTime
+  - Temporal.TimeZone.name -> Temporal.TimeZone.id
+  - Temporal.YearMonth -> Temporal.PlainYearMonth
+  - Temporal.YearMonth.difference -> Temporal.PlainYearMonth.since
+  - Temporal.YearMonth.isLeapYear -> Temporal.PlainYearMonth.inLeapYear
+  - Temporal.YearMonth.minus -> Temporal.PlainYearMonth.subtract
+  - Temporal.YearMonth.plus -> Temporal.PlainYearMonth.add
+  - Temporal.YearMonth.toDateOnDay -> Temporal.PlainYearMonth.toPlainDate
+- New APIs:
+  - Temporal.Calendar.fields
+  - Temporal.Duration.blank
+  - Temporal.Duration.round
+  - Temporal.Duration.total
+  - Temporal.Instant.toZonedDateTime
+  - Temporal.Instant.toZonedDateTimeISO
+  - Temporal.Instant.until
+  - Temporal.now.plainDateISO
+  - Temporal.now.plainDateTimeISO
+  - Temporal.now.zonedDateTime
+  - Temporal.now.zonedDateTimeISO
+  - Temporal.PlainDate.until
+  - Temporal.PlainDateTime.toZonedDateTime
+  - Temporal.PlainDateTime.until
+  - Temporal.PlainTime.until
+  - Temporal.PlainYearMonth.until
+  - Temporal.ZonedDateTime
+- Removals:
+  - Temporal.Calendar.dateMinus
+  - Temporal.DateTime.toInstant
+  - Temporal.Instant.toDateTime
+- Conversion methods that add information, such as
+  Temporal.PlainYearMonth.toPlainDate(), now always take a property bag
+  with the fields to add.
+- Temporal.Instant.toString() now takes one argument, an options bag.
+  The time zone may be specified as the value of the `timeZone` property
+  in this argument.
+- The toString() methods of Temporal.PlainDate, Temporal.PlainDateTime,
+  Temporal.PlainMonthDay, Temporal.PlainTime, and
+  Temporal.PlainYearMonth now take a `calendar` property which controls
```

Comment:
You recently renamed the time zone equivalent of this property to `timeZoneName`, correct?  Seems like this should be renamed to `calendarName` for consistency. I can PR this.

Reply:
PR is ready if it'd be welcome

Reply:
Sure, go for it.

Reply:
Done. #1138